examples/medplum-scheduling-demo/src/pages/UploadDataPage.tsx

```diff
@@ -135,11 +138,34 @@ function checkBotsUploaded(medplum: MedplumClient): boolean {
   const exampleBots = bots.filter(
     (bot) =>
       bot.name &&
-      ['book-appointment', 'cancel-appointment', 'set-availability', 'block-availability'].includes(bot.name)
+      ['book-appointment', 'cancel-appointment', 'set-availability', 'block-availability', 'example-data'].includes(
+        bot.name
+      )
   );
 
-  if (exampleBots.length === 4) {
+  if (exampleBots.length === 5) {
     return true;
   }
   return false;
 }
+
+async function uploadExampleData(medplum: MedplumClient, profile: Practitioner): Promise<void> {
+  // Unlike the other bundles, which source data from JSON files, this example data bundle is
+  // provided by a bot to handle dynamic content.
+  const batch: Bundle = await medplum.executeBot({ system: 'http://example.com', value: 'example-data' }, profile);
+  const result = await medplum.executeBatch(batch);
```

Comment:
That's the code comment that explains why we are not using a static JSON file for the example data of this demo app.